### DISCUSSION TITLE
arm64 version

### DISCUSSION THREAD

**Discussion by rfeg** [aurelienpierreeng/ansel#96] [General]
Hi Aurelien,

I think I found an away to please a big part of 9% share of mac os users, at least to help you test and validate your fork: can you compile for arm64 ubuntu version?

This can be also be a work around for mac users to test it (yes, part of 9%); I managed to install ubuntu on M1 mac.

Keep up the good work (that I would like to see and test).

Note: this comment is part of mac discussion, but I think this request is more general than Apple Silicon Macs, allow running on SoC computers, for _probably_ minimum effort. 

Regards,
rfeg

**Comment by aurelienpierre** [comment]
Nothing is minimum effort when it includes Mac. The CI Mac bot breaks every next week due to package issues.

**Comment by rfeg** [comment]
Thanks for your answer. I was mentioning ARM compilation, with that I could run on Ubuntu (on M1 mac).

Regards,
reg

**Comment by aurelienpierre** [comment]
FWIW Github doesn't provide ARM runners for workflows, so I have no way of building on ARM64 through Github actions and no way of delivering ARM64 AppImage until they/Microsoft Azure enable ARM 64 support. See https://github.com/actions/runner-images/issues/5631

**Comment by rfeg** [comment]
Hi Aurelien,

Sorry to ping you again, but is there any chance that you have progressed
in making Ansel available for ARM or do you have a tentative date?

Thank you and keep up the good work.
Thanks,
rfeg
